src/RightsService.Data.Provider.MsSql.Ef/Migrations/20220120201000_UpdateRightsLocalizationTable.cs

```diff
@@ -0,0 +1,42 @@
+﻿using LT.DigitalOffice.RightsService.Models.Db;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace LT.DigitalOffice.RightsService.Data.Provider.MsSql.Ef.Migrations
+{
+  [Migration("20220120201000_UpdateRightsLocalizationTable")]
+  [DbContext(typeof(RightsServiceDbContext))]
+  public class UpdateRightsLocalizationTable : Migration
+  {
+    protected override void Up(MigrationBuilder migrationBuilder)
+    {
+      migrationBuilder
+        .UpdateData(
+        table: DbRightsLocalization.TableName,
+        keyColumns: new[] { "RightId", "Locale" }, 
+        keyColumnTypes: new []
+        {
+          "int",
+          "string",
+        },
+        keyValues: new object[] { 10, "ru" },
+        columns: new[] { "Name", "Description" },
+        columnTypes: new[] { "string", "string" },
+        values: new[] { "Удаление и добавление сотрудников департамента", "Право позволяет удалять и добавлять сотрудника департамента в системе" });
+
+      migrationBuilder
+        .UpdateData(
+        table: DbRightsLocalization.TableName,
+        keyColumns: new[] { "RightId", "Locale" },
+        keyColumnTypes: new[]
+        {
+          "int",
+          "string",
+        },
+        keyValues: new object[] { 10, "en" },
+        columns: new[] { "Name", "Description" },
+        columnTypes: new[] { "string", "string" },
+        values: new[] { "Removing and adding department users", "This right allows you to add and remove department user in the time system" });

```

Comment:
Name: department data management 
Description: This right allows you to add, modify and remove department user in the time system; to add, modify and remove projects and news in the department this right is attached to